### DISCUSSION TITLE
Please change Rustdesk so that the EXE is no longer detected as a virus or Trojan

### DISCUSSION THREAD

**Discussion by mmvisualgit** [rustdesk/rustdesk#6261] [General]
RustDesk version V1.1.9 has not been classified as a virus by any virus scanner.

However, version V1.2.2 and V1.2.3 seem to contain viruses or Trojans.

Please change the Windows EXE so that no virus is detected with https://www.virustotal.com.

Thank you very much for help. Best Regards Markus.

**Comment by Kambaa** [comment]
What can the application developers do if other tools mark this application? I have no idea, it seems somebody scammed and RustDesk gets the bad end of things. What a shame!


![resim](https://github.com/rustdesk/rustdesk/assets/5601326/cc2eb218-11e3-4b23-af0f-baadf2ccd877)

**Comment by subscribernamegoeshere** [comment]
by going through the windows installer of current rustdesk just briefly  and observing what it does and how select features are being programmed, it comes to my mind that rustdesk uses this cmd/commandline way of executing scripts and bringing in its own root certificate for that one catalog file of the driver/dll layers and stuff that handle the display part of things etc. its installer is signed by a real authenticode executable signature purchased by some vendor, but that one file at least comes with a self signed kind of root-certificate like installed component into the users windows system. that looks like a developer way or debugging way to do stuff

of couse certificates in the windows world are often being sold for huge money and sums and you need to submit driver level stuff etc to microsoft certification procedures and all kinds of stuff you need to enroll in. maybe this could be enhanced in the future and kind of made a bit in a more official software release way rather than these kind of hacky ways ;)

it would be also great if rustdesk was really properly documented how e.g. the installer works, what steps it takes and where it puts what files, what locations, files registry it uses to store select things and what network level resouces it connects to, bootstraps from etc for example for people be able to judge the software and to follow through with analysis and build up trust.

thanks.
